mozilla-release/mobile/android/base/java/com/cliqz/SystemAddon.java

```diff
@@ -0,0 +1,49 @@
+package com.cliqz;
+
+import org.mozilla.gecko.EventDispatcher;
+import org.mozilla.gecko.util.EventCallback;
+import org.mozilla.gecko.util.GeckoBundle;
+
+public class SystemAddon {
+    private SystemAddon() {
+
+    }
+
+    public static void sendTelemetry(GeckoBundle signal, String schema) {
+        final GeckoBundle signalBundle = new GeckoBundle();
+        signalBundle.putBundle("data", signal);
```

Comment:
```java
private static GeckoBundle wrap(GeckoBundle bundle) {
    final GeckoBundle bundle = new GeckoBundle();
    bundle.putBundle("data", bundle);
    return bundle;
}

private static GeckoBundle wrap(String string) {
    final GeckoBundle bundle = new GeckoBundle();
    bundle.putBundle("data", string);
    return bundle;
}
```

Instead of repeating the same 2 lines over and over.